addon/components/rdfa/editor-toolbar.hbs

```diff
@@ -45,7 +45,7 @@
         </span>
       </button>
       {{/if}}
-      {{#if @showListButtons}}
+      {{#if (and @showListButtons canInsertList)}}
```

Comment:
Please make sure to properly scope variables in templates. It should be `this.canInsertList` in ember octane.